service/model/release.go

```diff
@@ -50,6 +50,15 @@ type Release struct {
 	UpdatedAt    time.Time
 	GitTagName   string
 	GitTagURL    url.URL
+	Attachments  []ReleaseAttachment
+}
+
+type ReleaseAttachment struct {
+	ID        uuid.UUID
+	Name      string
+	FilePath  string
+	URL       url.URL
+	CreatedAt time.Time
 }
```

Comment:
- To keep things simple, there is simple `ReleaseAttachment` struct. 
- If the app needs to store more types of files in the future, it would be a good opportunity to refactor and introduce a general `File model` (IMHO).
- Although `FilePath` could serve as a natural identifier, I thing it is more flexible and practical to use UUID as ID.

Note to FilePath field:
_Since the service model doesn't concern itself with where the files are stored, I chose the name FilePath for the field. While cloud storage services often use terms like 'storage object key' or 'storage object ID,' I believe FilePath is more general and does not imply that a specific type of storage is being used_